docs/REST API/rest-api.md

```diff
@@ -6936,12 +6938,74 @@ Deletes the session specified by ID. When the user associated with the session n
 
 ## Software
 
+- [Upload a software](#upload-a-software)
+- [Delete a software](#delete-a-software)
 - [List software titles](#list-software-titles)
 - [List software versions](#list-software-versions)
 - [Count software](#count-software)
 - [Get software title](#get-software-title)
 - [Get software version](#get-software-version)
 
+### Upload a software
```

Comment:
```suggestion
### Upload software
```

No "a" needed here. Also, do we use "Upload" elsewhere in the docs? I would make sure we're being consistent with endpoints that are similar like scripts (add v. upload). Do we use "Add a script" or "Upload a script"?

Reply:
@noahtalerman  I've also changed this for Scripts in the same PR, to match what we're doing here.